jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.py

```diff
@@ -78,12 +78,22 @@ def __init__(self, *args, **kwargs):
 
         self.results_label_default = 'Sonified data'
         self.add_to_viewer_selected = 'flux-viewer'
+        self._results_label_counter = 1
 
     @property
     def user_api(self):
         expose = ['sonify_cube']
         return PluginUserApi(self, expose)
 
+    @observe('results_label_invalid_msg')
+    def _avoid_temporary_vue_error(self, event):
+        """
+        Update `results_label` when a new sonification is added to the viewer to avoid
+        an unsightly vue error. This also updates the label with a new 'default'.
+        """
+        if self.results_label in self.app.data_collection and self._avoid_temporary_error:
+            self.results_label = f"{self.results_label_default} ({self._results_label_counter})"
```

Comment:
we actually want to set `results_label_default` so that if the user had entered something manually it does not get overwritten.